connector_search_engine/wizards/se_binding_state_updater.py

```diff
@@ -14,7 +14,12 @@ class SeBindingStateUpdater(models.TransientModel):
         selection=lambda self: self.env["se.binding"]._fields["state"].selection,
         required=True,
     )
+    do_it_now = fields.Boolean(help="Don't wait for the cron to process these records")
 
     def doit(self):
         res_ids = self.env.context.get("active_ids")
-        self.env["se.binding"].browse(res_ids).write({"state": self.state})
+        bindings = self.env["se.binding"].browse(res_ids)
+        bindings.write({"state": self.state})
+        if self.do_it_now:
+            domain = [("id", "in", bindings.ids)]
+            bindings.index_id.generate_batch_recompute_per_index(domain=domain)
```

Comment:
I just drafted this quickly and in fact is not working because the domain is meant to be for indexes. I'm working on a proper implementation.